ext/odbc/php_odbc.c

```diff
@@ -397,87 +402,6 @@ PHP_MINIT_FUNCTION(odbc)
 	le_pconn = zend_register_list_destructors_ex(NULL, _close_odbc_pconn, "odbc link persistent", module_number);
 	odbc_module_entry.type = type;
 
-	REGISTER_STRING_CONSTANT("ODBC_TYPE", PHP_ODBC_TYPE, CONST_CS | CONST_PERSISTENT);
```

Comment:
Apparently, `PHP_ODBC_TYPE` is only defined for win 32 at the top of the file:
```

#ifdef PHP_WIN32
#include <winsock2.h>

#define ODBC_TYPE "Win32"
#define PHP_ODBC_TYPE ODBC_TYPE

#endif
```

How come it works for other platforms? Or is ODBC Windows-only?

Reply:
This macro is defined in [php_odbc_includes.h](https://github.com/php/php-src/blob/master/ext/odbc/php_odbc_includes.h) for other platforms. It seems reasonable to move the Windows specifc definition to that file as well.